PodTemplates.yaml

```diff
@@ -27,6 +27,26 @@ spec:
     securityContext:
       privileged: false
     tty: true
+  - args:
+    - "99d"
+    command:
```

Comment:
Is the sleep a left behind from debugging?

Reply:
It's just a way to be sure that the container will be running for the lifetime of the job.
The main container process run 'sleep 99d' and the the jenkinsfile can run as many sh'' steps, this has to be done for every container in a pod

Reply:
Jenkins is then responsible to cleanup the pods once done